### ISSUE TITLE
Cannot preprocess files with other extensions than .svelte

### ISSUE THREAD

**Issue by MarcCoet** [jakedeichert/svelvet#63]
We can't use [MDsveX](https://github.com/pngwn/MDsveX) for example because of [this](https://github.com/jakedeichert/svelvet/blob/master/src/index.ts#L72)

**Comment by jakedeichert** [comment]
According to [mdsvex's docs](https://github.com/pngwn/MDsveX#use-it), you can use whatever extension you want.

>// whatever file extension you want to use has to go here as well
> extensions: ['.svelte', '.svexy', '.svx'], // here actually

What extensions are you using?

**Comment by MarcCoet** [comment]
True. I don't know if there are other preprocessors that use other file extension either...
I currently use `.md` because it gave me easy syntax highlighting but my mind is not settled (in another project I use `.svx`).

You know that I am using a modified copy of svelvet so don't worry about it if it is just me. I just guess someone else will end up having this problem at some point.

When using MDsveX with Sapper, we need to do `sapper export --ext '.md .svelte'`. Maybe that is a good idea to be able to specify which files need to be processed like svelte (or JS) files and maybe even which ones need to be copied as is or simply ignored.

I am not holding my breath so don't hesitate to wait for other feedback.

**Comment by jakedeichert** [comment]
Well adding an option like `--extensions svelte,svx,md` could make sense here, to get the behavior you want. Any supplied extensions would be processed be the svelte compiler.
